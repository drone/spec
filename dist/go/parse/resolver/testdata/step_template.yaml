kind: pipeline
spec:
  stages:
  - type: ci
    spec:
      steps:
      - type: template
        spec:
          name: golang
          inputs:
            version: 1.19
      - type: plugin
        spec:
          name: slack
          inputs:
            channel: engineering
  - type: template
    spec:
      name: node
      inputs:
        version: 16
