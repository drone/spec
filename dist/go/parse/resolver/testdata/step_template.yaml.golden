kind: pipeline
spec:
  stages:
  - type: ci
    spec:
      steps:
      - type: script
        spec:
          image: golang
          run: |-
            go build
            go test
      - type: script
        spec:
          image: plugins/slack  
  - type: ci
    spec:
      steps:
      - type: script
        spec:
          image: node
          run: |-
            npm install
            npm run test
                
            